# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by scripts/Create-CopyPatch.
# 
# T2 SDE: package/*/grub2/lvm2-hotfix.patch
# Copyright (C) 2021 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Somehow my cached raid volumes throw invalid segment errors, so let's patch this out
for now as this code is too yolo for me to futher debug.

  - Rene Rebe <rene@exactcode.de>

--- grub-39cfb3eb5caa71967f4e9e741d859e15d645c32f/grub-core/disk/diskfilter.c.vanilla	2021-03-12 22:21:50.747395643 +0100
+++ grub-39cfb3eb5caa71967f4e9e741d859e15d645c32f/grub-core/disk/diskfilter.c	2021-03-12 22:22:43.543392496 +0100
@@ -528,7 +528,7 @@
   grub_err_t err;
 
   if (seg->stripe_size == 0 || seg->node_count == 0)
-    return grub_error(GRUB_ERR_BAD_FS, "invalid segment");
+    ; // return grub_error(GRUB_ERR_BAD_FS, "invalid segment");
 
   switch (seg->type)
     {
